blockchain/state_migration.go

```diff
@@ -302,6 +309,7 @@ func (bc *BlockChain) PrepareStateMigration() error {
 	}
 
 	bc.prepareStateMigration = true
+	logger.Info("State migration is prepared", "migrationStartingBlockNumber", bc.CurrentBlock().NumberU64()+1)
```

Comment:
I remembered the migration block number could be either `bc.CurrentBlock().NumberU64()` or `bc.CurrentBlock().NumberU64()+1`, depending on situation?

Reply:
Depending on a situation, migration block number could be `bc.CurrentBlock().NumberU64() + 1` or `bc.CurrentBlock().NumberU64() +2`.
This log is just logging for preparing. I didn't think this block number is important. 
Should we remove the block number? :)

I intended to log the prepare information.

Reply:
It would be good idea to do one of below :
1. remove blockNumber
```suggestion
	logger.Info("State migration is prepared", "migrationStartingBlockNumber")
```
2. write both blockNumber
```suggestion
	blockNumber := strconv.Itoa(bc.CurrentBlock().NumberU64()+1) + " or " strconv.Itoa(bc.CurrentBlock().NumberU64()+2)
	logger.Info("State migration is prepared", "migrationStartingBlockNumber", blockNumber)
```
3. change to expected
```suggestion
	logger.Info("State migration is prepared", "expectedMigrationStartingBlockNumber", bc.CurrentBlock().NumberU64()+1)
```

Reply:
Good. I like option 3. :)